client/templates/study_groups/study_group_online_members.js

```diff
@@ -0,0 +1,14 @@
+Template.studyGroupOnlineMembers.helpers({
+  onlineUsers: function(){
+    const onlineUsernames =  Meteor.users.find({ "status.online": true }, { fields: { username: 1 }})
+                        .fetch()
+                        .map(m => m.username);
```

Comment:
we should use _id, it will be globally-unique, `m._id`

Reply:
done